docs/source/models.rst

```diff
@@ -81,6 +81,141 @@ time_created
 run_id
     ID of the run that created the model, if the model was saved using :ref:`tracking`.
 
+signature
+  :ref:`model signature <model-signature>` in JSON format.
+
+input_example
+  reference to an artifact with :ref:`input example <input-example>`.
+
+
+.. _model-metadata:
+
+Model Metadata
```

Comment:
The term `Model Metadata` is somewhat vague (e.g., creation timestamp is metadata). Maybe `Model Inference Metadata` would be clearer?

Reply:
Alternatively, for more visibility, I suppose we could lift `Model Signature` and `Model Input Example` up one level and make them their own sections, rather than subsections.

Reply:
Yeah I agree Metadata is vague, but I would like to have signature and example under one section, not super important, but how about calling the section "Model Signature And Input Example"?